fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java

```diff
@@ -708,6 +710,7 @@ private HdfsFs getFileSystemByCloudConfiguration(CloudConfiguration cloudConfigu
             }
             return fileSystem;
         } catch (Exception e) {
+            Thread.interrupted();
```

Comment:
Can we try to unify the exception handler for all FileSystems

Reply:
later